hack/run-integration-tests.sh

```diff
@@ -40,4 +40,4 @@ tar xzf etcd.tar.gz
 export PATH=${PATH}:${PWD}/etcd-v3.5.9-linux-amd64
 rm etcd.tar.gz
 
-CONTROLLER_TEST_NUM_OBJECTS=12 go test -v ./test/integration/controller-manager -args -v=5
+CONTROLLER_TEST_NUM_OBJECTS=360 go test -v ./test/integration/controller-manager -args -v=5
```

Comment:
@MikeSpreitzer what's the difference in execution time between 12 and 360 ?

Reply:
Not a problem. You can see that the GitHub workflow ran in an acceptable amount of time.